after/syntax/javascript.vim

```diff
@@ -19,7 +19,18 @@ hi! link jsDocTypeBrackets         DraculaCyan
 hi! link jsFuncArgOperator         Operator
 hi! link jsFunction                Keyword
 hi! link jsTemplateBraces          Special
+hi! link jsModuleAsterisk          DraculaPurpleItalic
+hi! link jsVariableDef             DraculaPurple
 
-"}}}
```

Comment:
This part of the diff is just noise from a commit that removed fold-markers where it shouldn’t have—it would be nice to restructure the patch series to avoid that, if possible. A `rebase -i` could do it, with some care (I suggest working in a separate branch that starts on the same tip; makes it easier to experiment). 

The idea being to avoid noise both in the history and in the blame... this makes it easier to gather actual information from the commit history of a file.